### PR TITLE
Gidk 1f7ca3 custom ui changes| custom field design issue and bugs resolved

### PR DESCRIPTION
**Pull request by rohit-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

**Review by shalinee-walkover — APPROVED** [review]
Approve

File: apps/web-giddh/src/app/contact/contact.component.ts [hunk 1]
```diff
@@ -1383,6 +1385,7 @@ export class ContactComponent implements OnInit, OnDestroy, OnChanges {
         this.groupService.getCompanyCustomField().subscribe(response => {
             if (response && response.status === 'success') {
                 this.companyCustomFields$ = observableOf(response.body);
+                this.colspanLength = 11 + response.body.length;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/contact/contact.component.ts:1388]: Add nulity check

File: apps/web-giddh/src/app/shared/header/components/new-manage-groups-accounts/manage-groups-accounts.component.ts [hunk 2]
```diff
@@ -77,7 +80,7 @@ export class ManageGroupsAccountsComponent implements OnInit, OnDestroy, AfterVi
     public selectedRowIndex: number = null;
 
 	// tslint:disable-next-line:no-empty
-	constructor(private store: Store<AppState>, private groupWithAccountsAction: GroupWithAccountsAction, private formBuilder: FormBuilder, private cdRef: ChangeDetectorRef,
+    constructor(private store: Store<AppState>, private groupWithAccountsAction: GroupWithAccountsAction, private formBuilder: FormBuilder, private cdRef: ChangeDetectorRef,private _breakPointObservar: BreakpointObserver,
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/shared/header/components/new-manage-groups-accounts/manage-groups-accounts.component.ts:83]: remove _ from private class obj

File: apps/web-giddh/src/app/shared/header/components/new-manage-groups-accounts/manage-groups-accounts.component.ts [hunk 3]
```diff
@@ -105,6 +108,12 @@ export class ManageGroupsAccountsComponent implements OnInit, OnDestroy, AfterVi
 	// tslint:disable-next-line:no-empty
 	public ngOnInit() {
 
+        this._breakPointObservar.observe([
+            '(max-width:767px)'
+        ]).subscribe(result => {
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/shared/header/components/new-manage-groups-accounts/manage-groups-accounts.component.ts:113]: unsubscribe also

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/contact/contact.component.ts [hunk 4]
```diff
@@ -251,6 +251,7 @@ export class ContactComponent implements OnInit, OnDestroy, OnChanges {
     public currentPage: number = 1;
     /** company custom fields list */
     public companyCustomFields$: Observable<any[]>;
+    public colspanLength: number = 11;
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/contact/contact.component.ts:254]: Missing JS DOC

**Review by prateek-giddh — CHANGES_REQUESTED** [review]

**Review by prateek-giddh — APPROVED** [review]

**Review by ravinderwalkover — APPROVED** [review]

**Pull request merged**